### ISSUE TITLE
Training u2net using custom dataset

### ISSUE THREAD

**Issue by FatmaMazen2021** [He-jerry/U2Net-Tensorflow#2]
First of all,thank you for your well organized and memory friendly code.
Could you kindly tell me how to use your code for training u2net on the custom dataset?
What are the steps?
Thanks in advance

**Comment by He-jerry** [comment]
For the custom dataset setting, please check data.py;getTrainGenerator(),this function is to load the SOD images and masks;
And for the training path setting,please check u2net_train.py(Because I re-implement the U2Net and F3Net simultaneously, so it contains the F3Net training code);

**Comment by FatmaMazen2021** [comment]
Thank you for your fast response.
Should I clone your code first?
Could you kindly tell me the difference between U2NET and F3NET?
After modifying data.py and adding paths to my images and masks,should I use the following command?
Python train.py?
Are there any arguments to add to the command?
Thanks in advance.

**Comment by He-jerry** [comment]
Answer:
1. How could you run the code without clone the code?
2.The paper of both networks are available and both have their network name
in the title. They are two different networks using different strategy
(multiscale, subnetwork combination).
3.please check the training code Python file name carefully, the training
of U2Net is u2net_train.py
4.No external arguments because I just write it for SOTA comparison.

FatmaMazen2021 ***@***.***>於 2022年1月31日 週一，下午1:32寫道：

**Comment by FatmaMazen2021** [comment]
Ok sir 
Thank you very much
